nitro-protocol/hardhat.config.ts

```diff
@@ -73,7 +73,7 @@ const config: HardhatUserConfig & {watcher: any} = {
       chainId: 31415,
     },
     hyperspace: {
-      url: 'https://filecoin-hyperspace.chainstacklabs.com/rpc/v0',
+      url: 'https://api.hyperspace.node.glif.io/rpc/v1',
```

Comment:
This wasn't strictly necessary but it seemed simpler to use the same node endpoint for deployment